robot/Cumulus/tests/browser/donations_payments/create_donation_and_payment.robot

```diff
@@ -25,20 +25,24 @@ Create Donation from a Contact
     Click Special Related List Button  Payments    Schedule Payments
     Wait For Locator    frame    Create one or more Payments for this Opportunity
     Choose Frame    Create one or more Payments for this Opportunity
-    Enter Payment Schedule    ${No_of_payments}    ${intervel}    ${frequency}
     ${loc}    Get NPSP Locator    id    inputX
     Input Text    ${loc}    8/15/2018
+    Enter Payment Schedule    ${No_of_payments}    ${intervel}    ${frequency}
```

Comment:
For ${intervel}, is that supposed to be ${interval} ? I assume that 'intervel' is what someone coded in the app at some point in the past.

Reply:
its a variable that's storing the intervel value. It actually holds a number